docs/how_to/deploy/adreno.rst

```diff
@@ -214,28 +535,14 @@ We can choose from *float16*, *float16_acc32* (Mixed Precision), *float32* (stan
 To leverage the GPU hardware capabilities and utilize the benefits of half precision computation and memory management,
 we can convert an original model having floating points operation to a model operating with half precision.
 Choosing lower precision will positively affect the performance of the model, but it may also have a decrease in the accuracy of the model.
-To do the conversion you need to write a simple conversion function and specify the *dtype* value of "float16" before calling the function:
+
+To do the conversion you need to call adreno specific transformation API as soon relay module is generated through any frontend:
 
 .. code:: python
 
-   def  convert_to_dtype(mod, dtype):
-      # downcast to float16
-      if  dtype == "float16":
-         global  conv2d_acc = "float16"
-         from  tvm.ir  import  IRModule
-         mod = IRModule.from_expr(mod)
-         seq = tvm.transform.Sequential(
-            [
-                  relay.transform.InferType(),
-                  relay.transform.ToMixedPrecision()
-            ]
-         )
-         with  tvm.transform.PassContext(opt_level=3):
-            mod = seq(mod)
-      return  mod
-
-   dtype="float16"
-   mod = convert_to_dtype(mod["main"], dtype)
+   from tvm.relay.op.contrib import adreno
+   adreno.convert_to_dtype(mod["main"], "float16")
```

Comment:
this is syntax sugar, we need to mention here that tvm.relay.op.contrib.adreno.convert_to_dtype is implemented through ToMixedPrecision pass